code/datums/mutations/_mutations.dm

```diff
@@ -103,7 +105,7 @@
 	if(limb_req && !acquirer.get_bodypart(limb_req))
 		return TRUE
 	for(var/datum/mutation/human/mewtayshun as anything in acquirer.dna.mutations) //check for conflicting powers
-		if(!(mewtayshun.type in conflicts) && !(type in mewtayshun.conflicts))
+		if(!(mewtayshun.type in typesof(conflicts)) && !(type in typesof(mewtayshun.conflicts)))
```

Comment:
I feel like you should just update the relevant conflict lists rather than do this

Reply:
i don nwannaa

Reply:
All you have to do is do
```
/datum/mutation/mysnowflake/New()
    . = ..() 
    conflicts = typesof(/datum/mutation/meme)
```